runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -91,7 +91,7 @@ private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 					if (clanchatImage != null)
 					{
 						int width = clanchatImage.getWidth();
-						Point imageLocation = new Point(textLocation.getX() - width / 2, textLocation.getY() - clanchatImage.getHeight());
+						Point imageLocation = new Point(textLocation.getX() - width / 2 - 1, textLocation.getY() - clanchatImage.getHeight() - 1);
```

Comment:
Instead of simply substracting 1 pixel from Y position, center the icon on Y axis

Reply:
I don't think it affects anything, because you are measuring font height, not rendered text or string height, so it will be always on same position, no matter what characters in name there would be.

Reply:
Ok. Had to remove the max descent because it is included in `getFontHeight` and it would cause what I mentioned before (even when you don't have these letters in your name).